docker-compose-dev.yml

```diff
@@ -0,0 +1,30 @@
+version: "3.3"
+services:
+  near-node:
+    image: rainbowup-image
+    build: .
+    network_mode: host
+    command: bash -c "node index.js start near-node; node index.js start near-node && while true; do sleep 30; done;"
+  ganache:
+    image: rainbowup-image
```

Comment:
It's better to specify dockerimage, so it builds based on a local dockerfile:
https://docs.docker.com/compose/compose-file/#dockerfile

Reply:
This still should build from the local image. This should just ensure we don't rebuild it several times, but once.

Reply:
No, if you `docker-compose build` it's smart enough to build it once

Reply:
Also `docker-compose build` would be a bit more common practice than `docker build . -t rainbowup-image`

Reply:
so, you mean if I just specify build: Dockerfile everywhere that should work?

Reply:
Yeah. we used this way in explorer: https://github.com/near/near-explorer/blob/master/docker-compose.yml

Reply:
Done.